saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/authentication/Saml2AuthenticationRequestContext.java

```diff
@@ -164,14 +231,44 @@ public Builder relayState(String relayState) {
 			return this;
 		}
 
+		/**
+		 * Sets the {@code RequestedAuthnContext} parameter that will accompany this
+		 * AuthNRequest.
+		 * @param requestedAuthnContext the RequestedAuthnContext value.
+		 * @return this object
+		 */
+		public Builder requestAuthnContextRefs(List<String> requestAuthnContextRefs) {
```

Comment:
Spring Security's builders tend to take a `Consumer` here instead of the list itself. That pattern allows for specifying a default list that the application can incrementally change.

Would you please change this to:

```java
public Builder requestAuthnContextRefs(Consumer<List<String>> requestAuthnContextRefsConsumer)
```

Reply:
That is an interesting partner did not know that 👍